src/test.ts

```
@@ -1,34 +1,36 @@
-const { Request } = require('superagent');
+import { inspect } from 'node:util';
+import { STATUS_CODES } from 'node:http';
+import { Server as HttpsServer } from 'node:tls';
+import type { Server, AddressInfo } from 'node:net';
+import { deepStrictEqual } from 'node:assert';
+import { Request, type Response } from 'superagent';
+import { AssertError } from './error/AssertError.js';
+
+export type TestApplication = Server | string;
+
+export type AssertFunction = (res: Response) => AssertError | void;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)